get_test.go

```diff
@@ -753,6 +753,10 @@ func TestGetFile_inplace_badChecksum(t *testing.T) {
 	if err == nil {
 		t.Fatalf("err is nil")
 	}
+	_, ok := err.(*ChecksumError)
+	if ok == false {
```

Comment:
It's conventional to condense this if it's not the variables aren't used elsewhere:

```suggestion
	if _, ok := err.(*ChecksumError); !ok {
```